proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/MetsUtils.java

```diff
@@ -108,8 +107,8 @@
  */
 public class MetsUtils {
 
-    private static Logger LOG = Logger.getLogger(MetsUtils.class.getName());
-    private static Properties mimeToExtension = new Properties();
+    private static final Logger LOG = Logger.getLogger(MetsUtils.class.getName());
+    private static final Properties mimeToExtension = new Properties();
```

Comment:
mimeToExtension by mělo být napsáno velkým písmem

Reply:
Já bych to nechal malým, není to konstanta, protože Properties je mutable.

Viz https://stackoverflow.com/a/12069687/889819